silk/model_factory.py

```diff
@@ -45,7 +45,7 @@ def default(self, o):
 
 
 def _parse_content_type(content_type):
-    """best efforts on pulling out the content type and encoding from Content-Type header"""
+    """best efforts on pulling out the content type and encoding from content-type header"""
     try:
         content_type = content_type.strip()
     except AttributeError:
```

Comment:
It seems this line is no longer covered, but I can't think of a situation where `content_type` would not be a string and thus would trigger an `AttributeError`, maybe we can remove this try-catch?